browsers/edge.json

```diff
@@ -30,11 +30,13 @@
         },
         "17": {
           "release_date": "2018-04-30",
-          "release_notes": "https://docs.microsoft.com/en-us/microsoft-edge/dev-guide",
+          "release_notes": "https://docs.microsoft.com/en-us/microsoft-edge/dev-guide/whats-new/edgehtml-17",
           "status": "current"
         },
         "18": {
-          "status": "nightly"
+          "release_date": "2018-10-01",
```

Comment:
I think the update started rolling out on October 2: https://blogs.windows.com/windowsexperience/2018/10/02/how-to-get-the-windows-10-october-2018-update/#WBYIOI2Lce9pjxmD.97